dice_maiden.rb

```diff
@@ -497,21 +513,21 @@ def check_bot_info(event)
             event.respond "#{@user} Roll #{@dice_result}"
             check_fury(event)
           else
-                  event.respond "#{@user} Roll: `#{@tally}` #{@dice_result}"
-                  check_fury(event)
+            event.respond "#{@user} Roll: `#{@tally}` #{@dice_result}"
+            check_fury(event)
           end
         end
       else
           if check_wrath == true
             event_comment_wrath(event, dnum)
           else
             if @simple_output == true
-        event.respond "#{@user} Roll #{@dice_result} Reason: `#{@comment}`"
-        check_fury(event)
-      else
-            event.respond "#{@user} Roll: `#{@tally}` #{@dice_result} Reason: `#{@comment}`"
-            check_fury(event)
-      end
+              event.respond "#{@user} Roll #{@dice_result} Reason: `#{@comment}`"
```

Comment:
Should this be two spaces or one?
```suggestion
              event.respond "#{@user} Roll #{@dice_result}  Reason: `#{@comment}`"
```